keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -236,11 +240,14 @@ Once the tests show that the mirrors are good, we can adapt the official documen
 
 #### Alpha -> Beta Graduation
 
+- [ ] [krel] creates and publishes deb and rpm packages of Kubernetes components
```

Comment:
Here, again, I think we need to add two more items:

```suggestion
- [ ] [krel] creates and deb and rpm packages of Kubernetes components
- [ ] Packages are signed
- [ ] Repository indices are updated after packages are copied to the repositories
```

I am intentionally:
a) Splitting signing to a separate item
b) Not adding implementation details of how we sign or how we copy+update

Reply:
Great feedback,  TYSM @puerco !